src/main/java/com/gooddata/visualizationObject/VisualizationObject.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (C) 2007-2017, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+
+
+package com.gooddata.visualizationObject;
+
+import com.fasterxml.jackson.annotation.*;
+import static com.fasterxml.jackson.annotation.JsonTypeInfo.As.WRAPPER_OBJECT;
+import static com.fasterxml.jackson.annotation.JsonTypeInfo.Id.NAME;
+import static com.gooddata.visualizationObject.CollectionType.*;
+import static java.util.stream.Collectors.toList;
+
+import com.gooddata.executeafm.UriObjQualifier;
+import com.gooddata.executeafm.afm.*;
+import com.gooddata.md.AbstractObj;
+import com.gooddata.md.Meta;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@JsonTypeName(VisualizationObject.NAME)
```

Comment:
this is probably redundant

Reply:
without this, deserilizer would look for key `VisualizationObject` but on post, webapp receives json with key `visualizationObject`, thus it would not deserialize. Is there better way to achieve it?